bool taxi
geometry_msgs/Point point
